### PR TITLE
fix: fix bug not displaying asset messages correctly

### PR DESCRIPTION
**Pull request by gongracr**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

This PR contains the fix in kalium needed to show correctly on AR, asset messages sent by web clients 

Needs releases with:

- [x] GitHub link to other pull request 
https://github.com/wireapp/kalium/pull/413

#### How to Test

Log in on your web client account, try to send an asset message to a given conversation. Open the AR app, check that the asset placeholder gets displayed correctly on the same conversation

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [x] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [x] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-518/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3133138)(3133138)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3133138)(3133138)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt [hunk 1]
```diff
@@ -273,11 +273,6 @@ class ConversationViewModel @Inject constructor(
 
     private suspend fun mapToMessageUI(assetContent: AssetContent, conversationId: ConversationId, messageId: String): MessageContent {
         with(assetContent) {
-            val assetId = remoteData.assetId
-
-            // TODO: To be changed once the error behavior has been defined with product
-            if (assetId.isEmpty()) return TextMessage(messageBody = MessageBody("The asset message could not be downloaded correctly"))
```
Comment (saleniuk) [inline app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt:279]: so downloading assets isn't implemented yet and the same with handling the situation when `assetId` is empty, right? 🤔

Comment (gongracr) [reply app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt:279]: Exactly. Until we don't implement the downloading behavior, this is not relevant. But yes, once we do, we will need to figure out how to address the situation where the asset ID might be empty in between those 2 messages

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-518/5/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3139995)(3139995)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3139995)(3139995)-dev-release.apk)

**Review by saleniuk — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [6](https://10.10.124.134/job/wire-android-reloaded/job/PR-518/6/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3141048)(3141048)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3141048)(3141048)-dev-release.apk)

**Pull request merged**